tracking/src/DataCollection.js

```diff
@@ -0,0 +1,86 @@
+const NOT_AVAILABLE = "not available"
+class DataCollection {
+    constructor() {
+        console.log("DataCollection init"); 
+
+        this.getPlatform(); 
+        this.getLanguage(); 
+        this.getOS(); 
+        this.getScreenResolution(); 
+        this.getAppName(); 
+    }
+
+    toJson() {
```

Comment:
@swVisenze also need to capture:

- current page URL ("url" parameter)
- referrer ("referrer" url)
This is under section 5.1.1 last part.

Reply:
added url and referrer